dts/bindings/ethernet/qca,ar8031.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: Qualcomm AR8031 PHY
+
+compatible: "qca,ar8031"
+
+include: phy.yaml
+
+properties:
+  reg:
```

Comment:
most likely we should also put this in the phy.yaml but I didn't check all the drivers so not requirement for this PR

Reply:
removed as it is already included in phy.yaml